lib/data/external/utils.js

```diff
@@ -15,6 +20,67 @@ const utils = {
     removeQuotes(word) {
         return word.slice(1, -1);
     },
+    // Before putting or deleting object on Azure, check if MPU exists with
```

Comment:
are we covered in initiate mpu and object copy as well? i think we are by virtue of blocking the data putting but just want to confirm.